application/models/User_model.php

```diff
@@ -468,6 +470,8 @@ function update_session($id, $u = null, $impersonate = false) {
 			'user_default_band'	 => $u->row()->user_default_band,
 			'user_default_confirmation'	 => $u->row()->user_default_confirmation,
 			'user_qso_end_times' => isset($u->row()->user_qso_end_times) ? $u->row()->user_qso_end_times : 1,
+			'user_timezone' => $this->getTimezone($u->row()->user_timezone),
+			'user_show_log_utc' => isset($u->row()->user_show_log_utc) ? $u->row()->user_show_log_utc : 1,
```

Comment:
You stored the option `user_show_log_utc` in the users table directly. This is not necessary as there is a table called `user_options` available which can store dynamic options like this.